tree/dataframe/src/RNTupleDS.cxx

```diff
@@ -170,7 +170,7 @@ class RNTupleColumnReader : public ROOT::Detail::RDF::RColumnReaderBase {
 
 public:
    RNTupleColumnReader(RNTupleDS *ds, RFieldBase *protoField) : fDataSource(ds), fProtoField(protoField) {}
-   ~RNTupleColumnReader() = default;
+   ~RNTupleColumnReader() override = default;
```

Comment:
It is a bit odd that it is the only override needed ....

Reply:
Well, it has a `final` keyword already in GetImpl below.

Whereas Connect and Disconnect do not exist in RColumnReaderBase